packages/core/src/shared/telemetry/vscodeTelemetry.json

```
@@ -154,6 +154,36 @@
             "type": "boolean",
             "description": "true if user has selected code snippet, false otherwise."
         },
+        {
+            "name": "cwsprChatHasProjectLevelContext",
+            "type": "boolean",
+            "description": "true if query has project level context."
+        },
+        {
+            "name": "cwsprChatProjectLevelContextQueryLatencyInMs",
```

can say the same thing with less verbosity:

```suggestion
            "name": "cwsprChatProjectContextQueryMs",
```